bank.rb

```diff
@@ -0,0 +1,53 @@
+class Banking
+
+	def initialize(current_balance, deposit_amount, withdraw_amount)
+		@current_balance = current_balance
```

Comment:
Here, I'd probably just initialize with a balance and then put the deposit/withdrawal amount in their corresponding methods, like this:
```
def deposit(amt)
	@balance += amt
end
```